Gopkg.toml

```diff
@@ -30,19 +30,19 @@ required = [
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "~6.0"
+  revision = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.9.9"
+  version = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.9.9"
+  version = "kubernetes-1.11.0"
 
 [[override]]
   name = "github.com/json-iterator/go"
-  version = "1.1.4" # as used by apimachinery@kubernetes-1.9.9
+  version = "1.1.5" # as used by apimachinery@kubernetes-1.11.0
```

Comment:
How did you determine that apimachinery uses 1.1.5 at the kubernetes-1.11.0 tag? From the Godeps file (below) it looks like it still uses a commit that's contained in 1.1.4:

https://github.com/kubernetes/apimachinery/blob/103fd098999dc9c0c88536f5c9ad2e5da39373ae/Godeps/Godeps.json#L94-L95

That said, I'm fine updating to 1.1.5 if it works. We should just reword the comment to say `# same minor version track as used by apimachinery@kubernetes-1.11.0`

Reply:
@enisoc How do you get the info that the commit is 1.1.4? The relevant PR should be [this](https://github.com/json-iterator/go/commit/f2b4162afba35581b6d4a50d3b8f34e33c144682), but tag 1.1.5 is in bold.

Reply:
I see - we should take the earliest (not latest) tag. Thank you!
I verify that 1.1.5 works.